app/DynamicsAdapter/DynamicsAdapter.Web.Test/Mapping/MappingProfileTest.cs

```diff
@@ -251,7 +357,7 @@ public void PersonSearchCompleted_should_map_to_SSG_SearchApiEvent_correctly()
             Assert.AreEqual(new DateTime(2003, 3, 3), searchEvent.TimeStamp);
             Assert.AreEqual(Keys.EVENT_COMPLETED, searchEvent.EventType);
             Assert.AreEqual(Keys.SEARCH_API_EVENT_NAME, searchEvent.Name);
-            Assert.AreEqual("Auto search processing completed successfully. 2 identifier(s) found.  2 addresses found. 2 phone number(s) found.", searchEvent.Message);
+            Assert.AreEqual("Auto search processing completed successfully. 2 identifier(s) found.  2 addresses found. 2 phone number(s) found. 1 employment(s) found.", searchEvent.Message);
```

Comment:
add name messages here.

Reply:
The PR is for employment data. So did not look at name